src/main/java/org/mitre/synthea/export/FhirR4.java

```diff
@@ -768,6 +806,64 @@ private static BundleEntryComponent encounter(Person person, BundleEntryComponen
     return entry;
   }
 
+  private static BundleEntryComponent navigationalAssistance(
```

Comment:
This should be removed. It has to do with some CODI stuff and isn't relevant to mainline Synthea.